src/builtins.ts

```diff
@@ -519,6 +520,8 @@ function isinstance(t: Thread, f: IPy_FrameObj, args: IPy_Object[], kwargs: Py_D
   var obj = args[0],
     cls = args[1];
 
+  if (obj === undefined || cls === undefined)
```

Comment:
Actually this is not needed. I was previously having issues in COMP_OP opcode where one of these would be undefined, but that no longer happens. (This was due to me creating exception object instances in LOAD_GLOBAL opcode, instead of type objects)